mflowgen/full_chip/constraints/cons_scripts/clocks.tcl

```diff
@@ -0,0 +1,222 @@
+#-----------------------------------------------------------------------------
+# TCL Script
+#-----------------------------------------------------------------------------
+# Purpose: Clocks
+#------------------------------------------------------------------------------
+#
+# Author   : Gedeon Nyengele
+# Date     : May 14, 2020
+#------------------------------------------------------------------------------
+
+
+# ------------------------------------------------------------------------------
+# Clock Parameters
+# ------------------------------------------------------------------------------
+set   master_clk_period               2.0
+set   alt_master_clk_period           1.0
```

Comment:
What do these clk periods correspond to? How would we adjust TLX vs SoC vs CGRA clocks?